tests/unit/OfficeAddresses.spec.ts

```diff
@@ -201,8 +202,10 @@ describe('summary mode', () => {
     // verify labels
     const labels = summaryRow.findAll('label')
     expect(labels.at(0).text()).toBe('Registered Office')
-    expect(labels.at(1).text()).toBe('Mailing Address Corrected')
-    expect(labels.at(2).text()).toBe('Delivery Address Corrected')
+    expect(labels.at(1).text()).toContain('Mailing Address')
+    expect(labels.at(1).text()).toContain('Corrected')
```

Comment:
I've seen some funky stuff happen calling this twice like this

Reply:
Really? Should work.